Source/Mosa.Compiler.Common/IntegerExtension.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) MOSA Project. Licensed under the New BSD License.
+
+namespace Mosa.Compiler.Common;
+
+public static class IntegerExtension
+{
+	public static string ToHex(this int i)
+	{
+		return $"0x{i:X}";
+	}
+
+	public static string ToHex(this long i)
+	{
+		return $"0x{i:X}";
+	}
+
+	public static string ToHex(this uint i)
+	{
+		return $"0x{i:X}";
+	}
+
+	public static string ToHex(this ulong i)
+	{
+		return $"0x{i:X}";
+	}
```

Comment:
Why not use the ``X4`` format for ``int`` and ``uint``, and ``X8`` for ``long`` and ``ulong``? It makes more sense in my opinion.

Reply:
I'd add different versions for fixed length (in future).